### ISSUE TITLE
Why the sync loss of given model is much bigger ?

### ISSUE THREAD

**Issue by cenalzw** [Rudrabha/Wav2Lip#113]
Thanks for your excellent work! I am trying to reproduce your work. I trained a syncnet by myself. But when I used my syncnet to supervise the generator network, I found the loss is much smaller than it from your lipsync_expert model (0.6 vs 6). So the syncnet got involved at the first iteration. I have no idea where the difference comes from. Is there anything wrong with my reproducing? Hope to see your help, Thanks a lot!

**Comment by prajwalkr** [comment]
What is the dataset you are training on?

**Comment by cenalzw** [comment]
@prajwalkr I use the official dataset LSR2.

**Comment by prajwalkr** [comment]
What is the accuracy of your model in detecting in-sync pairs and off-sync pairs ?

**Comment by cenalzw** [comment]
@prajwalkr The evaluation loss is about 0.68, I do not know if it is the accuracy you mean.

**Comment by prajwalkr** [comment]
The expert discriminator's eval loss should go down to ~0.25. Please train the expert discriminator until you reach this.

**Comment by cenalzw** [comment]
@prajwalkr I really appreacite your answer, thanks!. seems like it will take a long time for the syncnet to converge. I am just wondering if I need to decrease the learning rate during the training, because I saw the lr in your code is fixed.

**Comment by prajwalkr** [comment]
Yes, it does take a lot of steps to start converging. We are working to resolve this issue. But right now, it might take ~200k steps to start converging.

**Comment by DouYishun** [comment]
> @prajwalkr The evaluation loss is about 0.68, I do not know if it is the accuracy you mean.

@cenalzw @prajwalkr 
When I train the expert discriminator, I get 0.68 training and evaluation loss until 1000K steps.
